src/client/app/admin/user/user.class.ts

```diff
@@ -1,12 +1,9 @@
-import { Credentials } from './authentication/credentials.class';
+import { Credentials } from '../authentication/credentials.class';
 
 export class User {
-
-	constructor(
-		public userModel: any = {}, // raw User model object,
-		public credentials: Credentials = new Credentials(),
-		public eua?: any) {
-	}
+	public userModel: any = {}; // raw User model object,
```

Comment:
the convention we've generally been following is you don't need to explicitly use 'public' since it's the default